src/main/webapp/encounters/manualAnnotation.jsp

```diff
@@ -191,16 +193,16 @@ myShepherd.beginDBTransaction();
 
 try{
 	String vlist = "<p> 1. Select viewpoint: <select name=\"viewpoint\" class=\"notranslate\" onChange=\"return pulldownUpdate(this);\"><option value=\"\">CHOOSE</option>";
-	Query q = myShepherd.getPM().newQuery("javax.jdo.query.SQL", "select distinct(\"VIEWPOINT\") as v from \"ANNOTATION\" order by v");
-	List results = (List)q.execute();
+	final String noViewpoint = "----------";
+	vlist += "<option" + (noViewpoint.equals(viewpoint) ? " selected" : "") + ">" + noViewpoint + "</option>";
```

Comment:
i have checked our data, and it seems like the "unset" case will either be a `null` or `""` (empty string). so i think this logic here should be changed for that. (i.e. `viewpoint` will not be the `-------` value.)

and probably, for good luck, we should set a `value=""` (empty string) in there as well. a user picking that option will be setting the viewpoint to the dashed-line.

so maybe something like this for line 197:
```
vlist += "<option value=\"\"" + (((viewpoint == null) || viewpoint.equals("")) ? " selected" : "") + ">" + noViewpoint + "</option>";
```
(_if_ i got that all right!)

Reply:
good catch on the regex!  yeah this looks ready to go now. thanks!